src/pages/play.astro

```diff
@@ -54,113 +24,42 @@ changeLanguage('en')
               required
             />
           </div>
-          <button class='full-width' type='submit' disabled>
+          <button class='btn full-width' type='submit' disabled>
```

Comment:
The initial button had a padding of `0.75rem` and the font size is smaller as well. The previous one was more prominent and I think we were using the default font size for the button.

Reply:
i've bumped the font size back up. the previous button styles used the smaller text, but your point is good, buttons should be prominent.